Model/CustomConfigProvider.php

```diff
@@ -71,18 +71,18 @@ public function formatCheckoutData($checkoutData)
                 $data['paymentMethods'][] = [
                     'id'    => $method['subgroup'],
                     'value' => $method['group'] . ':' . $method['subgroup'],
-                    'name'  => $method['group'] == 'card' && $method['subgroup'] == 'card_input' && $this->config->get('header_name') ? $this->config->get('header_name') : $method['subgroup_title'],
+                    'name'  => $method['group'] == 'card' && $method['subgroup'] == 'card_input' && $this->config->get('checkout_title') ? $this->config->get('checkout_title') : $method['subgroup_title'],
                     'image' => $method['subgroup_logo']
                 ];
             }
-            if(count($data['paymentMethods']) <= 1 && $this->config->get('header_name'))
-                $data['paymentMethods'][0]['name'] = $this->config->get('header_name');
+            if(count($data['paymentMethods']) <= 1 && $this->config->get('checkout_title'))
```

Comment:
Un tema relacionado que vi por acá, la comparación tendría que ser de igual, porque si, por lo que sea, no llega ningún método va a saltar un error al acceder a la posición 0 y se va a romper la página del checkout.
```suggestion
            if(count($data['paymentMethods']) == 1 && $this->config->get('checkout_title'))
```